packages/terra-site/.npmignore

```diff
@@ -0,0 +1,12 @@
+*.log
+node_modules
+src
+aggregated-translations
+build
+stats.json
+lib/App.js
+lib/Home.js
+lib/index.html
+lib/Index.js
+lib/site.scss
+lib/TestLinks.js
```

Comment:
Do we want to add the terra-content, terra-form, terra-standout, and terra-title examples as well since they have been added as devDependencies?